test/deploy/deploy_unittest.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/bash
-git clone git@github.com:dmwm/deployment.git
-(cd deployment; curl https://github.com/dmwm/deployment/commit/3309be9ce3b628efe39e1db30b6a1f865e1fa3e4.patch | git am)
+WMA_TAG=1.0.7.pre3
+git clone https://github.com/dmwm/deployment.git
```

Comment:
command line options would be fine.

> On May 13, 2015, at 22:17, Alan Malta Rodrigues <notifications@github.com> wrote:
> 
> In test/deploy/deploy_unittest.sh <https://github.com/dmwm/WMCore/pull/5902#discussion_r30271245>:
> 
> > @@ -1,13 +1,17 @@
> >  #!/bin/bash
> > -git clone git@github.com:dmwm/deployment.git
> > -(cd deployment; curl https://github.com/dmwm/deployment/commit/3309be9ce3b628efe39e1db30b6a1f865e1fa3e4.patch | git am)
> > +WMA_TAG=1.0.7.pre3
> > +git clone https://github.com/dmwm/deployment.git
> Command line too?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dmwm/WMCore/pull/5902/files#r30271245>.
> 




Eric W. Vaandering 
Computing Physics Developer
CMS Computing Development Lead

Scientific Computing Division
Fermi National Accelerator Laboratory
630 962 7260 office
www.fnal.gov <http://www.fnal.gov/>